tsconfig.json

```diff
@@ -19,12 +19,12 @@
 		"noUnusedParameters": false,
 		"noUnusedLocals": false,
 
-		"sourceMap": true
+		"sourceMap": true,
 
-	},
-
-	"files": [
```

Comment:
You need to maintain the files list here for IntelliJ and presumably other tools to work properly.